src/schema/search.ts

```diff
@@ -385,10 +386,19 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         limit: limit.toString(),
         offset: offset.toString(),
       });
+      let meilieSearchRes = {};
+      if (args.version >= 3) {
+        meilieSearchRes = await mimirClient.search({
```

Comment:
Oh or is the mimir client just meili search wrapper?

Reply:
Yeah for now kinda is.
But ill check if viktor has protobuf already here